k8s-install/canal-kubernetes-datastore.yaml

```diff
@@ -181,3 +196,83 @@ spec:
         - name: flannel-cfg
           configMap:
             name: canal-config
+
+---
+
+apiVersion: rbac.authorization.k8s.io/v1beta1
+kind: ClusterRoleBinding
```

Comment:
You're binding the ClusterRole to a service account, but I don't see that service account actually passed to the Calico Pod at all.

See here: https://github.com/projectcalico/canal/blob/master/k8s-install/kubeadm/1.6/canal.yaml#L131